odl/phantom/misc_phantoms.py

```diff
@@ -146,6 +146,82 @@ def rect(x):
     return out.ufuncs.minimum(1, out=out)
 
 
+def text(space, text, font='arial', scale=0.8, inverted=True):
+    """Create phantom of text.
+
+    The text is represented by an scalar image taking values in [0, 1].
+    Depending on the choice of font, the text may or may not be anti-aliased.
+    anti-aliased text can take any value between 0 and 1, while
+    non-anti-aliased text takes values exclusively in {0, 1}.
+
+    This method requires the PIL package.
+
+    Parameters
+    ----------
+    space : `DiscreteLp`
+        Discretized space in which the phantom is supposed to be created.
+        Must be two-dimensional.
+    text : str
+        The text that should be written in the phantom.
+    font : str, optional
+        The font that should be used to write the text. Available options are
+        platform dependent.
```

Comment:
Any way to find out in pillow which ones there are? On a system level, the easiest way under Unix-like platforms is `find /usr/share/fonts -name "*.[to]tf"`, just to get some.